fast64_internal/oot/oot_utility.py

```diff
@@ -939,3 +939,41 @@ def getNewPath(type: str, isClosedShape: bool):
     bpy.context.view_layer.active_layer_collection.collection.objects.link(newPath)
 
     return newPath
+
+
+def updateTiedRoom(scene: Scene, emptyType: str, propName: str, ptrName: str):
+    """Used by handlers to update the tied room object
+
+    Parameters:
+    - ``emptyType``: the OoT empty type name (Water Box, Transition Actor, Entrance, ...)
+    - ``propName``: the properties name to get or set tied room objects
+    - ``ptrName``: the pointer property name
+    """
+
+    if scene.gameEditorMode != "OOT":
+        return
+
+    def updateTiedRoom(parentObj: Object, isScene: bool):
```

Comment:
please don't use the same function name twice

Reply:
oops lol didn't notice that